### DISCUSSION TITLE
Campaign briefing translation

### DISCUSSION THREAD

**Discussion by flegma83** [ihhub/fheroes2#8089] [General]
Hello.

When playing the original campaign, the briefing is in English instead of the Polish translation. Expansions have briefings in Polish.  
The remaining texts are displayed in Polish. How can I change the language of the basic campaign briefing?

Best regards!

**Comment by zenseii** [comment]
Hi, @flegma83! This is because we recently updated all these texts and for the Polish translation the changes haven't been verified yet. This is done by the community. It isn't hard to fix and doesn't require programming knowledge, but you have to know Polish to do this.

**Reply by flegma83** [reply]
Thank you for answer @zenseii
